recipes/classix/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: classix
+  version: 0.1.1
+
+
+source:
+  fn: ClassixClustering-0.5.8.tar.gz
+  url: https://files.pythonhosted.org/packages/6d/24/b1610d481857cf97815e34e366d7c384e4910a07b73510b7d952926699b6/ClassixClustering-0.5.8.tar.gz
```

Comment:
Please use the canonical url found in the `example/meta.yaml`:

https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L16